src/dataviews/dataview-model-base.js

```diff
@@ -117,6 +117,15 @@ module.exports = Model.extend({
     if (this.filter) {
       this.listenTo(this.filter, 'change', this._onFilterChanged);
     }
+
+    this.bind('error', function (model, response, options) {
+      if (response.status === 429) {
+        var error = response.responseJSON.errors_with_context[0];
+        setTimeout(function () {
```

Comment:
why we need to defer this?

Reply:
Because otherwise, the `error` event arrives after `error:_custom_` and could mess up the `listenTo` overriding the specific error with a generic one.

Reply:
It's kind a hack, and maybe it indicates a code smell. Is there any way to avoid this situation?

Reply:
The only way to avoid it would be not to trigger an error inside another error... or think about another way to communicate with Builder without triggering errors...

Reply:
A stack of errors?